### PR TITLE
Updated Introductory Section of the MCMC page

### PR DESCRIPTION
**Pull request by anetasie**
Corrected a few words in the Introductory text.

### CODE REVIEW

**Comment by DougBurke** [comment]
@anetasie - we also have similar information in

https://sherpa--1303.org.readthedocs.build/en/1303/mcmc/sim.html
https://sherpa--1303.org.readthedocs.build/en/1303/mcmc/mh.html

as this is docstring text in the `sherpa.sim` and `sherpa.sim.mh` files. We should probably simplify this (ie rely on the page you are editing to provide the overview and so reduce the amount of text in those two modules). Can you review them as part of this work?

**Comment by anetasie** [comment]
@DougBurke Yes, I agree that these docstrings need to be simplified. I'll work on them and include in this PR.

**Comment by anetasie** [comment]
@DougBurke index.rst file is ready for review. I did not deleted any text from the docstrings in the original `sherpa.sim` file.

**Comment by anetasie** [comment]
@DougBurke I finished editing the index.rst.  I did not delete any text from the docstrings in sim and mh files which should be done in a separate PR. 

This one is ready for the release, but needs a read and review.

**Review by hamogu — CHANGES_REQUESTED** [review]
Most of my comments are for sphinx formatting, others are typos. After you accept them, we'll have to check that the links all work - I may have typos in there.

If you agree to my suggestions, you can go to the file tab in the github pull request and click "add to batch" for those that you want to take. When you have all of them you can chose "commit" (https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-a-suggested-change). That way, you can apply them directly in GH and don't have to type them off again manually; it's probably the fastest method. Just remember to "git pull your_fork branch_name" to your local version if you need to do any local changes to your branch after, which seems unlikely since this is almost ready to merge.)

**Comment by anetasie** [comment]
@hamogu  I accepted the changes and 'add to batch' commit. I'll check the links when the RTD is ready. Thanks.

**Comment by anetasie** [comment]
@DougBurke @hamogu I fixed the links and checked that they work. There is of course more work on the individual docstrings, but I would work on the docstrings in a separate PR. This one should be ready.

**Review by hamogu — APPROVED** [review]

**Comment by hamogu** [comment]
I approved and removed @DougBurke pending review request - one review by SDS is sufficient for this work; so this is ready to be merged and could (from my point of view) still be included into 4.14, but it's not a disaster if it's not included either (in particular since this is a doc-only change and we currently default to showing the latest, not the last released version of the docs anyway).

File: docs/mcmc/index.rst
Comment (wmclaugh) [inline docs/mcmc/index.rst:29]:
```suggestion
[4]_. It can also account for instrument calibration uncertainty [5]_.
```

File: docs/mcmc/index.rst
Comment (wmclaugh) [inline docs/mcmc/index.rst:103]:
```suggestion
.. [1] https://hea-www.harvard.edu/AstroStat/pyBLoCXS/
```

**Comment by anetasie** [comment]
@wmclaugh I just updated the typos.

**Pull request merged**
